traffic_ops/traffic_ops_golang/crconfig/handler.go

```diff
@@ -271,11 +270,10 @@ func SnapshotOldGUIHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	cdn := inf.Params["cdn"]
-
 	// We never store tm_path, even though low API versions show it in responses.
 	crConfig, err := Make(inf.Tx.Tx, cdn, inf.User.UserName, r.Host, inf.Config.Version, inf.Config.CRConfigUseRequestHost, false)
 	if err != nil {
-		writePerlHTMLErr(w, r, inf.Tx.Tx, errors.New(r.RemoteAddr+" making CRConfig: "+err.Error()), err)
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, errors.New(r.RemoteAddr+" making CRConfig: "+err.Error()))
 		return
 	}
```

Comment:
If the user provides the name of a CDN that does not exist, I get a 500 error:

```
[user@host tmp]# curl -D- --cookie cookies https://${TO_HOST}.${INFRA_SUBDOMAIN}.${TLD_DOMAIN}/tools/write_crconfig/nonexistent
HTTP/2 500
access-control-allow-credentials: true
access-control-allow-headers: Origin, X-Requested-With, Content-Type, Accept, Set-Cookie, Cookie
access-control-allow-methods: POST,GET,OPTIONS,PUT,DELETE
access-control-allow-origin: *
content-type: application/json
set-cookie: mojolicious=...; Path=/; Expires=Tue, 09 Mar 2021 01:12:37 GMT; Max-Age=3600; HttpOnly
vary: Accept-Encoding
whole-content-sha512: /HT8WHLIsAIPgTPjZAa4GmpUwYtwph3fcU1//Cr7W4rkL1lKRW3kUEDTIz1DoA3/zUm3oa0elunc5VHcism6Qg==
x-server-name: traffic_ops_golang/
content-length: 62
date: Tue, 09 Mar 2021 00:12:37 GMT

{"alerts":[{"text":"Internal Server Error","level":"error"}]}
```

Errors that are the user's fault should be 400-level.